spec/lib/payment_processor/braintree/subscription_spec.rb

```diff
@@ -296,9 +299,33 @@ module Braintree
               end
             end
 
-            describe 'and subscription is successfully created' do
+            # The behvaiour tested by this block is already covered
+            # in request/api/braintree/braintree_spec.rb. This whole spec file
+            # is too dependent on stubs, and is too brittle to maintain.
+            # FIXME: Needs urgent refactoring/culling.
```

Comment:
This situation is largely a product of a less-experienced developer (me) taking the "unit test should test only the class in question, stub everything else" thing through to its logical conclusion. I'd love to hear what you guys think we should do for testing coordinator classes like this one - just integration specs?